radixdlt-java/src/main/java/com/radixdlt/client/application/translate/ApplicationStore.java

```diff
@@ -25,7 +24,6 @@ public ApplicationStore(
 			particleStore.getParticles(address)
 				.scanWith(reducer::initialState, reducer::reduce)
 				.debounce(1000, TimeUnit.MILLISECONDS)
-				.compose(ReplayingShare.instance())
```

Comment:
Remove this because don't want to cache state which is potentially old and invalid. This was originally used as the main mechanism to cache atoms but that's done in a different layer.

Reply:
So can we remove the dependency now, since this is the only place we needed it?

Reply:
Will remove it in next pull request